codespell_lib/data/dictionary.txt

```diff
@@ -23128,6 +23128,8 @@ resopnses->responses
 resorce->resource
 resorces->resources
 resore->restore
+resoruce->resource
+resoruces->resources
```

Comment:
Perhaps add resourced and resourcing equivalent typos too?

Reply:
I agree and added these typos for all existing `->resource` and `->resources`.